Source/WebCore/rendering/HitTestResult.cpp

```diff
@@ -413,7 +405,7 @@ URL HitTestResult::absoluteImageURL() const
         || is<HTMLInputElement>(*imageNode)
         || is<HTMLObjectElement>(*imageNode)
         || is<SVGImageElement>(*imageNode)) {
-        auto imageURL = imageNode->document().completeURL(downcast<Element>(*imageNode).imageSourceURL());
+        auto imageURL = imageNode->document().completeURL(uncheckedDowncast<Element>(*imageNode).imageSourceURL());
```

Comment:
Seems better to do a `dynamicDowncast<Element>` first and do all the `is<>` on that, and then not have an `uncheckedDowncast` here.